easybuild/easyconfigs/a/ARPACK/ARPACK96-goolf-1.5.14.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild config for ARPACK
+# Author: Bart Verleye, Center for eResearch, Auckland
+
+easyblock='MakeCp'
+
+name = 'ARPACK'
```

Comment:
Any reason why you want `ARPACK` instead of `arpack-ng`? If you just need the lapack/arnoldi algorithm, go for the `arpack-ng` package (already in EB).